src/browser/api/window.js

```diff
@@ -1002,6 +970,7 @@ Window.create = function(id, opts) {
             name
         });
     }
+    WebContents.setIframeHandlers(browserWindow.webContents, winObj, uuid, name);
```

Comment:
browserWindow.webContents => webContents , it's already defined before.

Reply:
It's defined > 500 lines above, I feel like browserWindow.webContents is more readable.